### ISSUE TITLE
Allow monospaced font for legends

### ISSUE THREAD

**Issue by gauthier-delacroix** [grafana/grafana#8304]
I'm using table legends a lot for series identified by a fixed length alpha-numeric ID.

It would be nice to be able to use a monospaced font for series name in a table legend so that all these IDs are aligned, and same for hover legend.

Here an example:
![6c3c788a-6884-48a5-97d8-e2d3df8d55e9](https://cloud.githubusercontent.com/assets/1533042/25746633/b25cae2e-31a4-11e7-8a13-18cfe0495997.png)

**Comment by MichaelSasser** [comment]
It would be great to see this implemented in tables in general. Not just in table legends.

**Comment by sylr** [comment]
Indeed, at the very least all numerical values in panels should use a monospace'd font.

Is it so frustrating to have to count digits to compare the scale of different values.

![image](https://github.com/grafana/grafana/assets/153052/158beca7-19ed-45d2-9059-3a93b7071f51)

**Comment by joshhunt** [comment]
Rather than using a whole monospace font, we could solve this by using Inter's (the font we use) tabular number support (which is essentially monospace but just for numbers)

https://github.com/grafana/grafana/assets/46142/f1e19bba-24b9-4aac-b18c-26d5ce7cecc2

I'll leave it with the @grafana/dataviz-squad folks to see what they think of this, and how broadly we could apply it.

**Comment by sylr** [comment]
@joshhunt suprisingly, `font-feature-settings: tnum` on the <body> applies to every numbers on page except the ones in the Y axis 🤔

**Comment by sylr** [comment]
@joshhunt is this being worked on ?

**Comment by erentar** [comment]
is there still no way to make numbers appear as monospace?

**Comment by SeWieland** [comment]
The issue is still open - I'd really like to use monospaced fonts in all my tables + legends... Is there any workaround yet?

**Comment by tajchert** [comment]
Hi, is there way to enable monospace font or tnum type on selected values/elements? This would be very helpful to improve readability of ex. values in tables.

**Comment by SilentGlasses** [comment]
I think similar to haw we have a Text Size option we should have a text format option, make it so it's only option is `Sans Serif` and `Monospace`...  Or at least use web safe font from Google Fonts like IBM Plex Mono for digits but only in panel value rendering, anything outside of a panel value can remain in Sans Serif.  Like your panel name can be Q1 2025 and that should be Sans Serif but the values in the panel should be Monospace.

**Comment by llmage** [comment]
Posting for visibility: My Team would also much appreciate a mono-space font option!

**Comment by sylr** [comment]
@joshhunt could you update us on this issue ?
